### PR TITLE
chore: added commitizen

### PR DESCRIPTION
**Pull request by Berkmann18**
Added an additional way to commit which would ensure that the commit message respects the Angular commit convention used by this repo.
_Usage_: `git cz` (I use an alias being `gz`) or `npm run commit`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,7 +13,8 @@
   "scripts": {
     "lint": "standard",
     "test": "ava --no-color test/*.js",
-    "semantic-release": "semantic-release"
+    "semantic-release": "semantic-release",
+    "commit": "git cz"
```
Comment (RichardLitt) [inline package.json]: Doesn't this depend upon having this alias in your .gitconfig?

Comment (Berkmann18) [reply package.json]: Perhaps but from what I know and experienced it's set automatically (just running `npm i -g comitizen` handles that).
I might be wrong tho but after having installed it on WSL on my work laptop, I didn't had to configure anything.

Comment (RichardLitt) [reply package.json]: Seems to work. You should add a note to the Contribution guide about what this is and how it works. Could you?

Comment (Berkmann18) [reply package.json]: Sure, I'll do that now.

**Comment by RichardLitt** [comment]
Thanks!

**Pull request merged**